### DISCUSSION TITLE
Installation question

### DISCUSSION THREAD

**Discussion by clickresults-za** [mschmidtkorth/alfred-clickup-msk#27] [Q&A]
Hi there
I am not sure on how to set this up.

Brand new to Alfred. Been using Clickup for quite a few months now, and it is time to streamline it :-)

I have downloaded Alfred, and bought the PowerPack license.

I have downloaded last version of your code.

I don't see the .alfredworkflow file though, and I also don't know how to "import" your code into Alfred to setup the new workflow.

Am I missing something?

**Comment by mschmidtkorth** [comment]
Hi @clickresults-za - you're going to have lots of fun with Alfred!

For the `.alfredworkflow` file please check out the [releases](https://github.com/mschmidtkorth/alfred-clickup-msk/releases) - the latest release will have the file to be downloaded (just double click it once it is on your machine). You do not need to download the source code, but feel free to do so - after all, the `.alfredworkflow` file is just a container.

The workflow will update itself if any new releases are made available.

**Comment by clickresults-za** [comment]
Thanks, this helped!
Eben Lingenfelder
Founder at Click Results
E  eben@clickresults.co.za
W  https://clickresults.co.za/
<https://clickresults.co.za/?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>

M 0766406339
[image: facebook]
<https://www.facebook.com/ClickResults/?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>
[image:
linkedin]
<https://za.linkedin.com/in/eben-lingenfelder?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>
Websites that drive your business forward
<https://clickresults.co.za?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>
<https://bookme.name/ebenlingenfelder/30-minute-zoom-call?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>
Click
to schedule a meeting
<https://bookme.name/ebenlingenfelder/30-minute-zoom-call?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>
<https://storysites.co.za?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>
Just
launched: Learn more about StorySites.co.za
<https://storysites.co.za?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>

**Comment by alexandrelissowski** [comment]
Hello, 
Thank you for developing such an integration between Alfred and Clickup. 

I used Clickup for my personal use since few time and recently I use Alfred to powerup my mac usage. 
I would like to simply create tasks from anywhere from my mac using Alfred, when I work. 

I succeeded to download and install your ClickUp workflows. It appears in Alfred but I unable to configure it. I don't know where to put the variable written in https://github.com/mschmidtkorth/alfred-clickup-msk#configuration. 
Could you describe the steps to follow in order to add them in Alfred? 

I use Alfred 5.1 and Clickup 3.2
MacOS Ventura 13.4. 

Thanks a lot

**Comment by mschmidtkorth** [comment]
Hi,  if you open Alfred and enter "cu:config" 
(without quotes) then hit Return, it should bring up the configuration menu. 

Alexandre Lissowski schrieb um 19:11 am 2023-05-19:

> 
> 
> 
> Hello,
> Thank you for developing such an integration between Alfred and Clickup.
> 
> 
> I used Clickup for my personal use since few time and recently I use Alfred to powerup my mac usage.
> I would like to simply create tasks from anywhere from my mac using Alfred, when I work.
> 
> 
> I succeeded to download and install your ClickUp workflows. It appears in Alfred but I unable to configure it. I don't know where to put the variable written in https://github.com/mschmidtkorth/alfred-clickup-msk#configuration.
> Could you describe the steps to follow in order to add them in Alfred?
> 
> 
> I use Alfred 5.1 and Clickup 3.2
> MacOS Ventura 13.4.
> 
> Thanks a lot
> 
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/mschmidtkorth/alfred-clickup-msk/discussions/27#discussioncomment-5950930>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AKX3POKBOTCDPRCHA37YXATXG6SVNANCNFSM4Y2V3ICQ>.
> You are receiving this because you commented.Message ID: ***@***.***>
> 

---
Beste Grüße
Michael Schmidt-Korth

**Comment by alexandrelissowski** [comment]
Hello @mschmidtkorth, 
Thank you for your quick answer. 
Unfortunately, after uninstalling Alfred and removing the workflow. 
Then, reinstalling everything, I still have the same issue. 
See what I have when I start typing `cu:confi` 
![Screenshot 2023-05-22 at 09 05 32](https://github.com/mschmidtkorth/alfred-clickup-msk/assets/54275025/bbd18025-1fd2-4092-be28-9d1e5bd0e1f0)

And here what I have when I fully type `"cu:config` 
![Screenshot 2023-05-22 at 09 05 40](https://github.com/mschmidtkorth/alfred-clickup-msk/assets/54275025/eb66f6d5-1c33-4bcf-9f9a-e33189a662cf)

Thank you for your help, and tell me if you need more details. 
Have a good day, 
Alex

**Comment by WreckItR** [comment]
Hi Alex,

Did you make sure you are using Phyton2?
https://github.com/mschmidtkorth/alfred-clickup-msk/issues/35

**Reply by WreckItR** [reply]
And can you run the debug in Alfred?

**Reply by alexandrelissowski** [reply]
Hello @RvDelft, 
Yes I can run it. 
I get this error message 
``` [11:50:25.746] ERROR: ClickUp[Script Filter] Code 127: /Users/alexandre/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/497AE2A8-EE85-4357-A4CC-41490B793D23: line 1: /usr/bin/python: No such file or directory``` 

Does it come from my version of Python? 
![Screenshot 2023-05-22 at 11 57 37](https://github.com/mschmidtkorth/alfred-clickup-msk/assets/54275025/e48b0319-047c-4c10-8823-0724d86d767d)


Thanks for your help, 
Alex

**Reply by WreckItR** [reply]
Hi Alex,
Yes, at the moment you need to install Python2 and change some things. 
You should take a look at this https://github.com/mschmidtkorth/alfred-clickup-msk/issues/35. There is a discussion and some fixes on how you should do this for now.

**Reply by alexandrelissowski** [reply]
Thanks, I did all the changes but it seems they do not apply as the cache needs to be reloaded. 
It launched the `reload` command but it still doesn't work

**Reply by WreckItR** [reply]
Hm what error do you receive?

**Reply by alexandrelissowski** [reply]
Hello @WreckItR, 

I see this error in the bug tracking 

```Code 127: /Users/alexandre/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/CD1BD08B-809F-4508-8F34-7100AAC43DFF: line 1: /usr/bin/python: No such file or directory```

Thanks for your help
